h2o-probes.d

```diff
@@ -27,4 +27,6 @@ provider h2o {
     probe receive_request_header(uint64_t conn_id, uint64_t req_id, const char *name, size_t name_len, const char *value,
                                  size_t value_len);
     probe send_response_status(uint64_t conn_id, uint64_t req_id, int status);
```

Comment:
Would it be possible to rename this probe to `send_response`, so that the naming convention would be in align with `receive_request(|_header)`?

Reply:
For sure. Can I do this in a follow-up PR so that I can handle this as well? ced7e9a7b -- If you prefer that I do it now, I'll merge master on to this branch. Lemme know.

Reply:
Yeah I'd appreciate it if you can merge master (can be non-ff merge) and make the necessary changes, so that this PR would be doing two things: rename `send_response_status` to `send_response`, adding `send_response_headers`.

I prefer doing that because IIUC that's the intention of the PR; see https://github.com/h2o/h2o/pull/2194/commits/f998fdb80e0b5198cb823e2e36c058c3d989a026.

Reply:
Sure! I'll get on it :+1:

Reply:
Successfully renamed the probe in: 9bb6f2b5